src/Domain/CoordinateMaps/TimeDependent/Shape.hpp

```diff
@@ -0,0 +1,162 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+#include <memory>
+#include <optional>
+#include <string>
+
+#include "ApparentHorizons/TagsTypeAliases.hpp"
+#include "ApparentHorizons/YlmSpherepack.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/TimeDependent/ShapeTransitionFunctions.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+#pragma once
+
+/// \cond
+namespace domain::FunctionsOfTime {
+class FunctionOfTime;
+}  // namespace domain::FunctionsOfTime
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace domain::CoordinateMaps::TimeDependent {
+
+/*!
+ * \ingroup CoordMapsTimeDependentGroup
+ * \brief Given a point with cartesian coordinates \f$\xi^i\f$, let the polar
+ * coordinates \f$(r, \theta, \phi)\f$ with respect to a center \f$x_c^i\f$ be
+ * defined in the usual way: \f{align}{
+ * \xi^0 - x_c^0 &= r sin(\theta) cos(\phi)\\
+ * \xi^1 - x_c^1 &= r sin(\theta) sin(\phi)\\
+ * \xi^2 - x_c^2 &= r cos(\theta)
+ * \f}
+ * The shape map distorts the distance \f$r\f$ between the point and the center
+ * while leaving the angles \f$\theta\f$, \f$\phi\f$ between them preserved by
+ * applying a spherical harmonic expansion with time-dependent coefficients
+ * \f$\lambda_{lm}(t)\f$. An additional domain-dependent transition function
+ * \f$f(r, \theta, \phi)\f$ ensures that the distortion falls off correctly to
+ * zero at the boundary of the domain. The monopole and dipole coefficients \f$
+ * \lambda_{00}, \lambda_{10}, \lambda_{11}\f$ are expected to be zero since
+ * these degrees of freedoms are controlled by the `SphericalCompression` and
+ * `Translation` map, respectively. The shape map maps the unmapped coordinates
+ * \f$\xi^i\f$ to coordinates \f$x^i\f$:
+ *
+ * \f{equation}{
+ * x^i = \xi^i - (\xi^i - x_c^i) f(r, \theta, \phi) \sum_{lm}
+ * \lambda_{lm}(t)Y_{lm}(\theta, \phi).
+ * \f}
+ *
+ * The inverse map is given by:
+ * \f{equation}{
+ * \xi^i = x_c^i + (x^i-x_c^i)*(r/\tilde{r}),
+ * \f}
+ * where \f$\tilde{r}\f$ is the radius of \f$\xi\f$, calculated by the
+ * transition map. For more details, see
+ * `TransitionFunc::original_radius_over_radius` in
+ * `Domain/CoordinateMaps/Protocols.hpp`.
+ *
+ * The frame velocity \f$v^i\ = dx^i / dt\f$ is calculated trivially:
+ * \f{equation}{
+ * v^i = x^i - (x^i - x_c^i) f(r, \theta, \phi) \sum_{lm}
+ * \dot{\lambda}_{lm}(t)Y_{lm}(\theta, \phi).
+ * \f}
+ *
+ * The Jacobian is given by:
+ * \f{equation}{
+ * \partial_j x^i = \delta_j^i \left( 1 - f(r, \theta, \phi) \sum_{lm}
```

Comment:
replace `\partial_j` with `\partial \xi^j`

Reply:
I dont understand, do you want me to do something like `\frac{\partial}{\partial \xi^j}`

Reply:
yes :)